jwcrypto/jws.py

```diff
@@ -30,7 +30,7 @@
     'RS256', 'RS384', 'RS512',
     'ES256', 'ES384', 'ES512',
     'PS256', 'PS384', 'PS512',
-    'EdDSA']
+    'EdDSA', 'ES256K']
```

Comment:
I think we shouldn't allow this by default until the draft is converted to an RFC.

Reply:
Fixed to fba5603.